modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -417,8 +417,6 @@ private void _initCustomObjectDefinition(
 		Dictionary<String, Object> properties =
 			HashMapDictionaryBuilder.<String, Object>put(
 				"companyId", companyIds
-			).put(
```

Comment:
Is it safe to remove this for all custom object definitions, or do we need some conditional logic here?